### DISCUSSION TITLE
Missions reset after disbanding island

### DISCUSSION THREAD

**Discussion by erik135bd** [BG-Software-LLC/SuperiorSkyblock2#1876] [Pending Feature Requests]
Is it already possible to automatically get all the mission reset after disband your island or leave/get banned from one? I still want the missions to be fixed to a player and not to an island, so with more island members every member got his own missions.

**Comment by OmerBenGera** [comment]
You can make the missions player-based. You can also make them reset upon disbanding the island.
All of this information can be found on the wiki.

**Comment by Lemvs** [comment]
@OmerBenGera you can close it.
